code/script.js

```diff
@@ -252,8 +304,16 @@ const setSecret = () => {
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
+  charactersInPlay = CHARACTERS;
+
+  //Restart - show the board again
+  board.style.display = "flex";
+
+  //hide winOrLose section
+  winOrLose.style.display = "none";
```

Comment:
The semi colons could be removed here, since you've decided to go without semi colons for the rest of the code.